<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDebugForm</class>
 <widget class="QWidget" name="UDebugForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Інформація виконання програми</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icontexto_webdev_pack/icontexto_webdev_pack/icontexto-webdev-info-032x032.png</normaloff>:/icontexto_webdev_pack/icontexto_webdev_pack/icontexto-webdev-info-032x032.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageDebug">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>537</width>
        <height>469</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="../images/UImgLib.qrc">
        <normaloff>:/UtechLibImg/libImages/utechLibImg_blue_speech_bubble_48.png</normaloff>:/UtechLibImg/libImages/utechLibImg_blue_speech_bubble_48.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Відладочні повідомлення (Debug)</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditDebugInfo">
         <property name="undoRedoEnabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageWarning">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>537</width>
        <height>469</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="../images/UImgLib.qrc">
        <normaloff>:/UtechLibImg/libImages/utechLibImg_comment_warning_48.png</normaloff>:/UtechLibImg/libImages/utechLibImg_comment_warning_48.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Зауваження (Warning)</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEditWarningInfo">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Вичистити вікто</string>
       </property>
       <property name="icon">
        <iconset resource="../images/UImgLib.qrc">
         <normaloff>:/UtechLibImg/libImages/utechLibImg_paper_48.png</normaloff>:/UtechLibImg/libImages/utechLibImg_paper_48.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_allWaysOnTop">
       <property name="text">
        <string>Вікно завжди поверх інших</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Закрити</string>
       </property>
       <property name="icon">
        <iconset resource="../images/UImgLib.qrc">
         <normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-cancel-032x032.png</normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-cancel-032x032.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/UImgLib.qrc"/>
 </resources>
 <connections/>
</ui>
